### PR TITLE
Update black version to 22.3.0

### PR DESCRIPTION
**Pull request by mircea-cosbuc**
Related to the CI failures: https://github.com/psf/black/issues/2964

### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Review by rodrigovalin — APPROVED** [review]
LGTM

File: .github/workflows/code-health.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Black Check
-        uses: jpetrucciani/black-check@20.8b1
+        uses: jpetrucciani/black-check@7f5b2ad20fa5484f1884f07c1937e032ed8cd939
```
Comment (irajdeep) [inline .github/workflows/code-health.yml:13]: Q: using a version here directly doesn't work?

**Review by irajdeep — APPROVED** [review]
LGTM

File: .github/workflows/code-health.yml (hunk 1 above)
Comment (mircea-cosbuc) [reply .github/workflows/code-health.yml:13]: There's no tag for the latest black release: https://github.com/jpetrucciani/black-check

**Pull request merged**